sentry/values.yaml

```diff
@@ -630,6 +649,15 @@ kafka:
   service:
     port: 9092
 
+  serviceAccount:
+    create: false
+    name: kafka
+
+  zookeeper:
+    serviceAccount:
+      create: false
+      name: zookeeper
```

Comment:
This changes the current behavior. The zookeeper chart will use the `name` even with `create: false`, so to use the current `default`, it needs to be overridden. Also for redis below, the other charts don't use the `name` if `create` is `false`.

Reply:
Sorry for that.
I created a new PR which adds comments around these configurations by default so it should reset the default behavour from before: https://github.com/sentry-kubernetes/charts/compare/develop...Phylu:patch-1

Reply:
Thanks!